src/lib/mapping/dispatch.ts

```diff
@@ -1,95 +0,0 @@
-import { Buffer } from "buffer";
-import { sign as signFunction } from "./sign";
-import { signature } from "./signature";
-import Transaction from "arweave/web/lib/transaction";
-import { createData, Signer } from "warp-arbundles";
-import { toBuffer } from "../utils/bufferUtils";
-import { userDetails } from "../auth/userDetails";
-
-/**
- * dispatch the given transaction. This function assumes (and requires) a user is logged in and a valid arweave transaction.
- * @param transaction The transaction to sign.
- * @returns The signed version of the transaction.
- */
-export async function dispatch(
-  transaction: Transaction,
-  node?: string,
-  arweave?: any,
```

Comment:
This was not really optional as the `catch` block below was assuming `arweave` would be defined. I think this might have been passed as a property due to some bundling issues I also encountered with the current setup when importing from `arweave` or `arweave/web`, but I've managed to do a quick fix for no at the top of `index.ts`.